docs/installation.md

```diff
@@ -35,49 +41,63 @@ This installation guide assumes that the following softwares are already install
 

```

Comment:
Are you saying that the listed prerequisites should be marked as dependencies? I believe they're not. Can you elaborate?